workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1103,71 +1103,101 @@ rule tip_frequencies:
             --proportion-wide {params.proportion_wide} \
             --output {output.tip_frequencies_json} 2>&1 | tee {log}
         """
-
-rule logistic_growth:
+# This analysis is deprecated as of 2025-01-23
+# This uses frequencies of strains in the tree to estimate logistic growth rates
```

Comment:
It would be safe to remove these commented lines, since we can recover them later from this PR and information in the change log. These changes aren't deprecations in the sense that they completely remove the functionality without a warning period where we discourage their use.

Reply:
Okay. This makes sense. I was keeping things around to be more easily findable without digging through Git. I've tried to thread the needle with 7bd4daa85a6debad344c6d6cfca4242f4356b9fc. This serves to document how the scripts were run.

Reply:
I see. This closed PR feels just as findable to me as an open issue. Thank you though.